doc/source/getting_started/XarrayforArviZ.ipynb

```diff
@@ -12,17 +12,17 @@
    "cell_type": "markdown",
```

Comment:
<p>I would not link to creating inference data here but do so below instead. I don't think people reading this explanation will expect to go to the creation tutorial at that point with that text.</p><p> </p><p>I would also add the general link to netcdf in the bullet points (same link as below in the netcdf section).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>